src/main/java/Duke.java

```diff
@@ -1,10 +1,90 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+//todo: modularise code
 public class Duke {
+    public static final int MAX_COMMANDS = 100;
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        Scanner sc = new Scanner(System.in);
+
+        boolean continueRun = true;
+        String userCmd = "";
+        String[] cmdArr = new String[MAX_COMMANDS];
+        int numOfTasks = 0;
+
+        ArrayList<Task> taskArrList = new ArrayList<>();
+
+        while (continueRun){
+            System.out.println("==========================");
+            System.out.println("What can I do for you?");
+            userCmd = sc.nextLine();
+
+            System.out.println("You have typed: " +userCmd);
+
+            //end program
```

Comment:
A one liner comment could be written this way:
// This is a single line comment